x-pack/plugins/infra/public/pages/metrics/hosts/components/host_details_flyout/metadata/table.tsx

```diff
@@ -54,12 +108,26 @@ export const Table = (props: Props) => {
   );
 
   return (
-    <EuiBasicTable
+    <EuiInMemoryTable
       data-test-subj="infraMetadataTable"
       tableLayout={'fixed'}
       responsive={false}
       columns={columns}
       items={rows}
+      search={search}
+      loading={loading}
+      error={searchError ? `Invalid search: ${searchError.message}` : ''}
```

Comment:
I'm wondering if the error message wouldn't be too long to display in the table. Or if we should just say that it failed to fech? Also, should we have a translation key for this?

Reply:
Yeah! IMO this is enough. `Invalid search:` was more explanatory though, would it be useful for some users?

Reply:
I used it before I put the message there, I think it's clear where the error is coming from so I don't think it's useful anymore.